### DISCUSSION TITLE
Tools from utils/*

### DISCUSSION THREAD

**Discussion by commonuserlol** [vfsfitvnm/frida-il2cpp-bridge#350] [General]
Hey, i saw you have some useful decorators/functions (e.g lazy, getter, decorate). Did you make them or did you get them from somewhere? Can I use them in my projects etc.

**Comment by vfsfitvnm** [comment]
Previously, I was using a third-party package instead of `lazy.ts`, but the way I reorganized the project didn't allow dependencies anymore, so I just included it manually. It's not rocket science though, you can get an identical result just by looking at stackoverflow.
The other ones are mostly created by me, but again it's not rocket science, they are based on common knowledge you can easily find online - feel free to use them
